### PR TITLE
docs: format sub-set of files to comply with template-1

### LINKED ISSUE #5: Update the Why files to align with the template
* update to new template
* go through for typos
* change to qmd?
* remove authors (#26) and make sure that dates stays as the modified date and not when the website was updated. See Decision #33

### PR DESCRIPTION
**Pull request by K-Beicher**
## Description

- This PR makes minor changes to formatting etc on a number of Why documents. The documents are:

_template
Containers - there will be a discussion post regarding relevance of this post
Cyclical Workflow - there is an issue to write this (#34)
Docker - there will be a discussion post regarding relevance of this post
Fly 
Github Flow

## Related Issues

Closes this is one PR of many from a single issue

## Reviewer Focus

This PR only needs a quick review.

## Checklist

For general documentation:

- [ x] Spell-check
    - [ ] US
    - [ ] UK
- [ x] Did the page(s) preview correctly on your machine without breaking
- [x ] New category words (keywords) (if any) added to the code snippet file

### CODE REVIEW

File: why-github-flow.qmd [hunk 1]
```diff
@@ -64,7 +63,7 @@ references:
       year: n.d.
     URL: https://githubflow.github.io
   - id: githubflow
-    title: GitHub flow - Follow GitHub flow to collaborateon projects.
+    title: GitHub flow - Follow GitHub flow to collaborate projects.
```
Comment (signekb) [inline why-github-flow.qmd:66]:
```suggestion
    title: GitHub flow - Follow GitHub flow to collaborate on projects.
```

File: why-github-flow.qmd [hunk 2]
```diff
@@ -1,8 +1,7 @@
 ---
 title: "Why GitHub Flow"
```
Comment (signekb) [inline why-github-flow.qmd:2]:
```suggestion
title: "Why GitHub flow"
```

File: why-cyclical-workflow.qmd [hunk 3]
```diff
@@ -1,11 +1,12 @@
 ---
 title: ""
```
Comment (signekb) [inline why-cyclical-workflow.qmd:2]:
```suggestion
title: "Why cyclical workflow"
```
Title needed here? :)

**Review by signekb — COMMENTED** [review]
Looks good! 
One question: Shouldn't the decision posts contain the same headers as the template? E.g., "Context and problem statement", "Decision drivers", "Considered options"?

File: why-github-flow.qmd [hunk 4]
```diff
@@ -1,8 +1,7 @@
 ---
-title: "Why GitHub Flow"
+title: "Why GitHub flow"
 description: "Our reasons for using the GitHub Flow as our branching and development strategy."
```
Comment (lwjohnst86) [inline why-github-flow.qmd:3]:
```suggestion
description: "Our reasons for using the GitHub flow as our branching and development strategy."
```

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**